zips/zip-0227.rst

```diff
@@ -282,6 +277,20 @@ It contains the following fields:
 The issuance bundle is added within the transaction format as a new bundle. 
 The detailed encoding of the issuance bundle as a part of the V6 transaction format is defined in ZIP 230 [#zip-0230-transaction-format]_.
 
+Computation of ρ
+----------------
+
+We define a function $\mathsf{DeriveIssuedRho} : \mathbb{F}_{q_{\mathbb{P}}} \times \{0 .. 2^{32} - 1\} \times \{0 .. \mathsf{MAX\_ISSUE}\} \to \mathbb{F}_{q_{\mathbb{P}}}$ as follows:
```

Comment:
Or better yet, use `(\mathbb{N}_0)` or `(\mathbb{N}^*)` (natural numbers with zero) for indexes

Reply:
The maximum value that can be issued is $2^{64} - 1$. So we should technically allow for an issuer to generate $2^{64} - 1$ notes to different parties with v=1 in each note. That was the rationale for using MAX_ISSUE ($= 2^{64} - 1$) over here.

Reply:
The index for the Actions is more complicated - we can have a very large number ($2^{4096}$ different 512-byte `asset_desc` strings possible) of different Assets issued by the issuer. 

So it doesn't really make sense to allow this whole range, so I reverted back to the usual way of encoding indices, which is with $\mathsf{I2LEOSP}_{32}$.

Reply:
We cannot use `\mathbb{N}` because when we need to encode, we need to use I2LEOSP to convert it to a sequence of bits, which needs the input argument to be constrained:
<img width="863" alt="image" src="https://github.com/user-attachments/assets/bd49ea00-c71b-455f-8dc4-631d70a198fb" />

Reply:
still, it is not practical to have $2^64-1$ notes in one issue transaction. Let's go for 32 bit for both.

Reply:
Done